Frontend/Pages/Shared/_LegalSummary.cshtml

```diff
@@ -0,0 +1,64 @@
+@using Frontend.ExtensionMethods
+@model Frontend.Models.LegalRequirements.LegalRequirementsViewModel
+<dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+                Trust agreement
+            </dt>
+            <dd class="govuk-summary-list__value">
+                @if (!Model.TrustAgreement.HasValue)
+                {
+                    <span class="dfe-empty-tag">Empty</span>
+                }
+                else
+                {
+                    <p class="govuk-body" data-test="trust-agreement">@(Model.TrustAgreement.ToDescription())</p>
+                }
+            </dd>
+            <dd class="govuk-summary-list__actions">
+                <a class="govuk-link" asp-page="/Projects/LegalRequirements/TrustAgreement" asp-route-urn="@Model.Urn" asp-route-returnToPreview="@(Model.ReturnToPreview ? "true" : null)" data-test="trust-agreemnt">
+                    Change<span class="govuk-visually-hidden"> trust agreement</span>
+                </a>
+            </dd>
+        </div>
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">
+                Diocesan consent
+            </dt>
+            <dd class="govuk-summary-list__value">
+                @if (!Model.DiocesanConsent.HasValue)
```

Comment:
Possible improvement to this would be to break out the common mark up into a helper function.
e.g. - https://github.com/DFE-Digital/academy-transfers-api/blob/d65834a2da5f42381c787abf36e7cacb9e88906a/Frontend/Views/Shared/_AdditionalInformation.cshtml#L53-L74

Reply:
Cool idea! have done it, would love your feedback but it definitely looks cleaner.